### ISSUE TITLE
Transpilation process became much slower since version 0.15.0

### ISSUE THREAD

**Issue by gal-indegy** [s-panferov/awesome-typescript-loader#83]
I use gulp.watch to monitor any change in my ts files and run webpack to bundle my scripts.
All used to work smooth up to awesome-typescript-loader 0.14.1.
Since I upgraded to awesome-typescript-loader 0.15.0, I couldn't work like that anymore, because every webpack run started taking ages (between 10 sec to 2 mins). Before this version it used to take a few seconds (1-5). Originally I've upgraded to 0.15.3, but then I was looking for the first version with the issue and got to 0.15.0.
I've tried to find what's the cause, but since I don't know the code I couldn't manage to find anything useful.

**Comment by s-panferov** [comment]
@gal-indegy please show me your config file. Maybe you faced with this problem: https://github.com/Swiip/generator-gulp-angular/pull/925

**Comment by gal-indegy** [comment]
Here's my config file:
module.exports = {
    entry: './src/scripts/main.js',
    resolve: {
        extensions: ['', '.webpack.js', '.web.js', '.ts', '.js']
    },
    devtool: 'source-map',
    module: {
        loaders: [
            { test: /\.json$/,  loader: 'json' },
            { test: /\.js$/,    loader: 'babel-loader', exclude: /(node_modules|bower_components)/ },
            { test: /\.ts$/,    loader: 'awesome-typescript-loader' },
            { test: /\.(png|jpg)$/, loader: 'url-loader?limit=8192' }
        ]
    }
};

**Comment by s-panferov** [comment]
@gal-indegy do you have `tsconfig` file?

**Comment by gal-indegy** [comment]
Oh sure.
```javascript
{
  "compilerOptions": {
    "target": "es5",
    "noResolve": false
  }
}
```

**Comment by s-panferov** [comment]
Please add `exclude` section to your config file as in the pull request above. Hmm, I think I need to add this exclude as default.

**Comment by gal-indegy** [comment]
Yes, that seems to make the process much better.
I agree it's a good idea to have this in as default, I guess others might face the same issue.
Thanks.

**Comment by s-panferov** [comment]
@gal-indegy resolved by #84
